benchmark.py

```diff
@@ -176,6 +177,35 @@ def RunBenchmark(args, logfile):
   else:
     print "Instance deletion is skipped due to --no-create_delete."
 
+  data_store_command = ("python store_data.py --ownip {}"
```

Comment:
I think it's best to import `store_data` as a module and invoke the Python methods directly, rather than invoke this as an external library. This will allow a tighter integration as needed between the two.